<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employeehomescreen</class>
 <widget class="QDialog" name="employeehomescreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1525</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1B2131;</string>
  </property>
  <widget class="QGroupBox" name="tcEmployeeProject">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>170</y>
     <width>1531</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tcEmployeeProject{border: treansparent; 
background-color: #1B2131;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>1421</width>
      <height>701</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(208, 208, 208);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tbAssigedProjects">
     <attribute name="title">
      <string>Assigned Porjects</string>
     </attribute>
     <widget class="QPushButton" name="btnStartProjcet">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>590</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:0px;
color: #EE5F8E;
border: 2px solid #EE5F8E;
background-color: transparent;</string>
      </property>
      <property name="text">
       <string>Start Project</string>
      </property>
     </widget>
     <widget class="QTableView" name="dgAssignedProjects">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1421</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tbCurrentProject">
     <attribute name="title">
      <string>Projects Inprogress</string>
     </attribute>
     <widget class="QPushButton" name="btnCompleteProject">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>590</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:0px;
color: #EE5F8E;
border: 2px solid #EE5F8E;
background-color: transparent;</string>
      </property>
      <property name="text">
       <string>Complete Project</string>
      </property>
     </widget>
     <widget class="QTableView" name="dgCurrentProjects">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1421</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tbCompletedProjects">
     <attribute name="title">
      <string>Completed Projects</string>
     </attribute>
     <widget class="QTableView" name="dgCompletedProjects">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1421</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="homeControlEmp">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>1211</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#homeControlEmp
{border: treansparent; 
background-color: #1B2131;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>10</y>
      <width>67</width>
      <height>67</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparentl</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../tms3Pics/InProgress.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>10</y>
      <width>67</width>
      <height>67</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparentl</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../tms3Pics/notStarted.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>37</y>
      <width>219</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_2{
border: 2px solid #797979;
border-radius:10px;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(121, 121, 121);
</string>
     </property>
     <property name="text">
      <string>Completed</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCompletedCount_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>836</x>
      <y>10</y>
      <width>67</width>
      <height>67</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparentl</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../tms3Pics/Completed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>270</y>
      <width>794</width>
      <height>332</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_4{
border: 2px solid #797979;
border-radius:20px;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="lblNameHeader_3">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>20</y>
       <width>341</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>David Mosby</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblRoleHeader_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(177, 177, 177);</string>
     </property>
     <property name="text">
      <string>Manager</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>411</width>
       <height>181</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="lblName_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #B7B7B9;</string>
      </property>
      <property name="text">
       <string>David Mosby</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblGender_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #B7B7B9;</string>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAddress_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #B7B7B9;</string>
      </property>
      <property name="text">
       <string>Lacinia Road, Lacinia </string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDepartment_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #B7B7B9;</string>
      </property>
      <property name="text">
       <string>Finance Department</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblRole_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #B7B7B9;</string>
      </property>
      <property name="text">
       <string>Manager</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblUsername_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #B7B7B9;</string>
      </property>
      <property name="text">
       <string>SandaruK</string>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>571</width>
       <height>7</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #AAAAAA;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="pUserAvatar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>193</width>
      <height>182</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparentl</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>37</y>
      <width>219</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{
border: 2px solid #797979;
border-radius:10px;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(121, 121, 121);
</string>
     </property>
     <property name="text">
      <string>Not Started</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNotStartedCount_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>37</y>
      <width>219</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_3{
border: 2px solid #797979;
border-radius:10px;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(121, 121, 121);
</string>
     </property>
     <property name="text">
      <string>In Progress</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblInProgressCount_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>label_15</zorder>
   <zorder>frame_4</zorder>
   <zorder>pUserAvatar</zorder>
   <zorder>frame</zorder>
   <zorder>frame_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_8</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>1525</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_3
{
border: rgb(26, 31, 68); 
border-bottom: 1px solid #878787;
border-top:1px solid #878787;
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HP Simplified</family>
      <pointsize>38</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>WorkMate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHome">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>130</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewProjects">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>10</y>
      <width>130</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Projects</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>10</y>
      <width>130</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>tcEmployeeProject</zorder>
  <zorder>homeControlEmp</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
